tests/ui/lint/dead-code/tuple-struct-field.rs

```diff
@@ -23,15 +28,19 @@ struct Void;
 struct GoodVoid(Void);
 
 fn main() {
-    let w = SingleUnused(42, [0, 1, 2, 3], "abc".to_string());
-    let _ = w.0;
-    let _ = w.2;
+    let u1 = UnusedAtTheEnd(42, 3.14, [0, 1, 2, 3], "def".to_string(), 4u8);
+    let _ = u1.0;
+
+    let _ = UnusedJustOneField(42);
+
+    let u2 = UnusedInTheMiddle(42, 3.14, "def".to_string(), 4u8, 5);
+    let _ = u2.0;
+    let _ = u2.3;
 
-    let m = MultipleUnused(42, 3.14, "def".to_string(), 4u8);
 
     let gu = GoodUnit(());
     let gp = GoodPhantom(PhantomData);
     let gv = GoodVoid(Void);
 
-    let _ = (gu, gp, gv, m);
+    let _ = (gu, gp, gv, u1);
```

Comment:
nit: it seems `u1` could be removed from `(gu, gp, gv, u1)`,  since it had been used in `let _ = u1.0`, is it right?

Reply:
You are right. I've removed it now.

Thanks.